es_test.go

```diff
@@ -1116,7 +1118,7 @@ func TestRegisterRepository_MissingType(t *testing.T) {
 
 	err := client.RegisterRepository(repo)
 
-	if err == nil || err.Error() != "Repository Type is required." {
+	if err == nil || err.Error() != "Repository Type is required" {
```

Comment:
Nit: seems like a good use for `errors.Is`

Reply:
@hoenn comment addressed.